primerpedia.css

```diff
@@ -113,3 +117,9 @@ object {
 	font-size: small;
 	color: gray;
 }
+
+@media (min-width: 600px) {
+    html {
+        font-size: calc(0.4vw + (13.6px * 1.45));
```

Comment:
Was this formula picked by any particular reason, or just by trial and error?

I would perhaps go to `13px + 0.5vw`, which makes the end result closer to the current behavior with flowtype, and is simpler and easier to reason about (in fact, I'll edit downstyle.css to use this, since the benefit of the lower slope at 0.4vw is not worth the extra complexity of the formula.)

Reply:
Sounds like a plan!